extensions/notification/destination/KafkaNotificationDestination.js

```diff
@@ -52,6 +52,8 @@ class KafkaNotificationDestination extends NotificationDestination {
             topic,
             pollIntervalMs,
             auth,
+            compressionType: 'none',
+            requiredAcks: '1',
```

Comment:
these params may need to be added to the destination config, may be done now or later, but need ticket either way (at product or at least zenko level, so that we can also track the remaining work: zkop, ...)